modules/stats/metrics/trades.py

```diff
@@ -50,3 +51,14 @@ def calculate_trade_durations(closed_trades: [Trade]):
     else:
         avg_trade_duration = longest_trade_duration = shortest_trade_duration = timedelta(0)
     return avg_trade_duration, longest_trade_duration, shortest_trade_duration
+
+
+def compute_median_trade_profit(closed_trades: [Trade]) -> float:
+    all_trade_profit = [trade.profit_dollar for trade in closed_trades]
+
+    if len(all_trade_profit) == 0:
+        return 0.0
```

Comment:
Do you think it would be better to switch these two? Not that this operation is computationally expensive - but I think it's just better practice to check beforehand.

```suggestion
    if len(closed_trades) == 0:
        return 0.0

    all_trade_profit = [trade.profit_dollar for trade in closed_trades]
```